nginx/kitspace.template

```diff
@@ -104,10 +100,21 @@ server {
 
 	location / {
 		proxy_intercept_errors on;
-		proxy_set_header    Host                $http_host;
-		proxy_set_header    X-Real-IP           $remote_addr;
-		proxy_set_header    X-Forwarded-For     $proxy_add_x_forwarded_for;
-		proxy_set_header    X-Forwarded-Proto   $scheme;
+		proxy_set_header    Host                          $http_host;
+		proxy_set_header    X-Real-IP                     $remote_addr;
+		proxy_set_header    X-Forwarded-For               $proxy_add_x_forwarded_for;
+		proxy_set_header    X-Forwarded-Proto             $scheme;
+		add_header          'Access-Control-Allow-Origin' '*' always;
+		add_header          'Access-Control-Allow-Methods' 'GET, OPTIONS';
+
+		if ($request_method = OPTIONS) {
+			add_header Content-Length 0;
+			add_header Content-Type text/plain;
+			add_header 'Access-Control-Allow-Origin' '*';
+			add_header 'Access-Control-Allow-Methods' 'GET, OPTIONS';
```

Comment:
Something I was wondering for a while, why are we adding these headers twice?

Reply:
Yes, the preflight methods are handled separately from other methods.

Reply:
Not really though. I think these lines are redundant. We are already adding the headers above.

Reply:
Have you tried just adding these two plus `return 200;` ? The rest of the seem to be repeating what's above. 

https://github.com/kitspace/kitspace-v2/blob/25cd4f45ee59183bf5f3e41e09b0bc73781bc119/nginx/kitspace.template#L82-L83

Reply:
But this will overwrite the response for non-options requests.
Aside, this `200` should be `201`.

Reply:
You are still adding headers twice for `OPTIONS`, once on line 107-108 and once on lines 113-114.

Reply:
Resolved via 081a257.